### PR TITLE
Update smp validation paths

### PR DESCRIPTION
**Pull request by npostiau**
@gpasztor (SMP trigger convener, for information)

Updated paths and histograms used by the HLT SMP validation (I am validator for HLT SMP):

- Monitoring of Single Electron triggers.
- Update of the paths monitored for Single Photon.

Changes have been tested by producing DQM files.

Will backport it in 9_2_X.

### CODE REVIEW

File: HLTriggerOffline/SMP/python/hltSMPValidator_cfi.py [hunk 1]
```diff
@@ -102,4 +98,15 @@
 	    # -- Analysis specific cuts
 	    minCandidates = cms.uint32(1), 
 	    ),
+    SingleEle = cms.PSet( 
+	    hltPathsToCheck = cms.vstring(
```
Comment (dmitrijus) [inline HLTriggerOffline/SMP/python/hltSMPValidator_cfi.py]: Don't use tabs, anywhere.

**Comment by dmitrijus** [comment]
Please introduce yourself and additionally send an email DQM core :)

**Comment by mtosi** [comment]
@cmsbuild, please test

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**